parsl/jobs/error_handlers.py

```diff
@@ -42,18 +43,25 @@ def _get_error(status: Dict[str, JobStatus]) -> Exception:
     err = ""
     count = 1
     for js in status.values():
+        err = err + f"Error {count}:\n"
+        count += 1
+
         if js.message is not None:
-            err = err + "{}. {}\n".format(count, js.message)
-            count += 1
+            err = err + f"\t{js.message}\n"
+
+        if js.exit_code is not None:
+            err = err + f"\tEXIT CODE: {js.exit_code}\n"
+
         stdout = js.stdout_summary
         if stdout:
             err = err + "\tSTDOUT: {}\n".format(stdout)
+
         stderr = js.stderr_summary
         if stderr:
             err = err + "\tSTDERR: {}\n".format(stderr)
```

Comment:
What's the likelihood of catastrophic failure?  Would it behoove the user for us to use `std(err|out)_summary` instead?

Reply:
not sure what you mean

Reply:
What I mean _now_ is that I was apparently blind earlier.  Now I clearly see:

```python
stdout = js.stdout_summary
...
stderr =  js.stderr_summary
```
I missed that first go-round.